pkg/kudoctl/cmd/diagnostics/collectors.go

```diff
@@ -0,0 +1,132 @@
+package diagnostics
```

Comment:
I would probably extract each collector in its own small file and put them all into the `diagnostics/collectors`. Wdyt?

Reply:
Generally: Yes :) But at the moment, the whole code is in a single package, this would require a full restructuring because of circular dependencies...